content/for-developers/tutorials/inner-loop/05-expose-app/expose-app.md

```diff
@@ -1,99 +1,165 @@
 # Expose your Application
 
-After successfully deploying your application on Red Hat OpenShift, you need to expose it to external traffic so that users can access your application over the network. This documentation will guide you through the process of exposing your application and making it accessible from outside the OpenShift cluster.
+Once you have successfully deployed your application on the SAAP (Stakater App Agility Platform), the next step is to make it accessible to users and resources over the network. In this tutorial, you will learn how to expose your application and enable external traffic to reach it, whether for internal use within the cluster or external access from outside the cluster. By following these steps, you will ensure that your application is reachable and can serve its intended purpose.
 
-## Route
+## Objective
+
+- Expose your application internally using the service
+- Expose your application externally using route or ingress
+
+## Key Results
+
+- Application is exposed for traffic in/out of the cluster
+
+## Tutorial #1
+
+### Exposing Your Application Within the Cluster
+
+Create a service:
+
+1. In your `deploy/values.yaml` file, define a service:
+
+    ```yaml
+    ## Service
+      service:
+        enabled: true
+        ports:
+        - name: http
+          port: 8080
+          targetPort: 8080
+    ```
+
+    It should look like this:
+
+    ![svc-values](images/svc-values.png)
+
+    a) Service is `enabled: true`, which means the service resource will be created on the SAAP when you tilt up your application.
+
+    b) `port:` This section defines the port configuration for the service resource.
+
+    c) `name: http`: We have seen in the route `targetPort: http` was given which was pointing to this service port named `http`. `port: 8080` is the port of the service itself. `targetPort: 8080` is the pod/container port of the application.
+
+      > You can change or add any configuration for the service. To see more configurations [click](https://github.com/stakater/application.git).
+
+1. Run `tilt up` at the root of your directory.
```

Comment:
change to: 

Run `tilt up` at the root of your directory. Open the tilt browser; just hit the space. If everything is green then the changes will be deployed in the cluster